blocks/team/team.css

```diff
@@ -1,4 +1,4 @@
 .team {
```

Comment:
This section is too close to the articles section, maybe add margin top to make it further

Reply:
Done